### PR TITLE
Spoiler tags

### PR DESCRIPTION
**Pull request by nahkampf**
Spoiler tags

- Reformatted every description to follow the new spoiler tag format
- Changed the contributing readme to reflect this standard

Cleanup
- The Xenogenesis books were out of their elements, put them under Xenogenesis entry
- Some typos (missing period, wrong formats etc).

Additions
- Becky Chambers "A psalm for the wild-built"
- Adrian Tchaikovsky "Children of time"

https://github.com/sindresorhus/awesome-scifi/issues/310

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -27,7 +27,13 @@ Please ensure your pull request adheres to the following guidelines:
...
 Commentary - [@you](https://github.com/you)
 
-> Description
+<details>
+  <summary>Description</summary>
+
+  > Summary of book/film etc here
+</details>
+
+*) Note the formatting (spaces etc) for "Description" above!
```
Comment (cassidoo) [inline contributing.md:36]:
```suggestion
* Note the formatting (spaces etc) for "Description" above!
```

Comment (cassidoo) [reply contributing.md:36]: Super small suggestion here!

File: readme.md [hunk 2]
```diff
@@ -51,25 +51,25 @@ Accelerando is an excellent exploration of Posthumanism. It's my go to recommend
...
 
-> The Singularity. It is the era of the posthuman. Artificial intelligences have surpassed the limits of human intellect. Biotechnological beings have rendered people all but extinct. Molecular nanotechnology runs rampant, replicating and reprogramming at will. Contact with extraterrestrial life grows more imminent with each new day.
->
-> Struggling to survive and thrive in this accelerated world are three generations of the Macx clan: Manfred, an entrepreneur dealing in intelligence amplification technology whose mind is divided between his physical environment and the Internet; his daughter, Amber, on the run from her domineering mother, seeking her fortune in the outer system as an indentured astronaut; and Sirhan, Amber's son, who finds his destiny linked to the fate of all of humanity.
->
-> For something is systemically dismantling the nine planets of the solar system. Something beyond human comprehension. Something that has no use for biological life in any form.
+<details>
+  <summary>Description</summary>
 
-##### [Adulthood Rites](https://www.goodreads.com/book/show/19105727-adulthood-rites) (1988) [4.2]
```
Comment (RichardLitt) [inline readme.md:60]: Why did you delete this one? Can you double-check all of these?

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (nahkampf) [reply readme.md:60]: They were out of sequence (Level 4 headings mixed in with Level 3 headings) and missing the standard info (author etc). There's an entry for the trilogy (Xenogenesis) elsewhere with that info, where they fit in. I figured that was the most correct way to handle them?

**Comment by RichardLitt** [comment]
Looks great to me. Thanks!
